packages/flutter/test/material/filled_button_test.dart

```diff
@@ -1,4 +1,4 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
+  // Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Might want to revert this...

Reply:
D'oh! Still not used to vscode's git interface and missed this in my review. Thx!